src/main/conversions/v2.21.0/c2_21_0_2018050101.clj

```diff
@@ -0,0 +1,44 @@
+(ns facepalm.c2-21-0-2018050101
+  (:use [kameleon.sql-reader :only [load-sql-file exec-sql-statement]]))
+
+(def ^:private version
+  "The destination database version."
+  "2.21.0:20180501.01")
+
+(defn- add-uniqueness-constraint
+  []
+  (exec-sql-statement
+   "ALTER TABLE ONLY interactive_apps_proxy_settings
+    ADD CONSTRAINT interactive_apps_proxy_settings_id_unique UNIQUE (id)"))
+
+(defn- drop-constraint
+  []
+  (exec-sql-statement
+   "ALTER TABLE ONLY interactive_apps_proxy_settings
+    DROP CONSTRAINT IF EXISTS interactive_apps_proxy_settings_container_settings_id_fkey"))
+
+(defn- remove-column
+  []
+  (exec-sql-statement
+   "ALTER TABLE ONLY interactive_apps_proxy_settings
+    DROP COLUMN IF EXISTS container_settings_id"))
+
+(defn- add-column
+  []
+  (exec-sql-statement
+   "ALTER TABLE ONLY container_settings
+    ADD COLUMN interactive_apps_proxy_settings_id uuid UNIQUE"))
+
+(defn- add-constraint
+  []
+  (load-sql-file "constraints/00_71_container_settings_fkey.sql"))
```

Comment:
If we move the `container_settings_interactive_apps_proxy_settings_id_fkey` constraint into the existing `constraints/71_container_settings.sql` file, then we'll have to update this function to manually add the new constraint, since reloading that file will probably cause an error trying to re-add the existing `container_settings_tools_id_fkey` constraint.